middleware/bussiness.js

```diff
@@ -0,0 +1,24 @@
+const User = require("../models/user")
+
+module.exports = async (req, res, next) => {
+    try {
+        const userId = req.userId
+
+        const loadedUser = await User.findOne({ _id: userId });
+        if (!loadedUser) {
+            const error = new Error("selected user not found");
+            error.statusCode = 404;
+            throw error;
+        }
+
+        if(loadedUser.role !== "BUSINESS") {
+            const error = new Error("user do not have bussiness privileges");
+            error.statusCode = 401;
```

Comment:
In my opinion, there should be 403 Forbidden however let @pawelsobkowski say what he thinks.

Reply:
Yes, it should be 403